### PR TITLE
Tlunizin 444

### PR DESCRIPTION
**Pull request by gsilver**
for 
https://jira.it.umich.edu/browse/TLUNIZIN-444

### CODE REVIEW

File: src/main/webapp/assets/js/utils.js [hunk 1]
```diff
@@ -403,6 +403,96 @@ $(document).on('click', '.openOtherInstructorModal', function (e) {
...
 
+
+
+// open a modal where user can search for courses with no instructor and select them
+$(document).on('click', '#courseStringTrigger', function (e) {
+  e.preventDefault();
+  var courseString = $.trim($('#courseString').val());
+  var termId = $.trim($('#canvasTermId').text());
+  
+  var mini='/manager/api/v1/accounts/1/courses?search_term=' + courseString + '&per_page=200';
```
Comment (lsloan) [inline src/main/webapp/assets/js/utils.js]: Is there no way to get this query path from sectiontool.properties so it doesn't need to be repeated here?

**Comment by lsloan** [comment]
Pretty good overall.  It seems like the user has to take an extra step to find the instructor-less courses, though.  Can't these courses just be included in the display by default?  If there tend to be many courses like that and it would make the display to crowded, hide the instructor-less ones by default in a section of the page that the user can expand.  Maybe I just feel like this because I haven't seen it in action.

File: src/main/webapp/assets/js/utils.js (hunk 1 above)
Comment (gsilver) [inline src/main/webapp/assets/js/utils.js]: Good point. We will make sure to do so in the next phase.

**Comment by gsilver** [comment]
Thanks again, Lance.  It is hard to tell in a slice that a code review is what the overall picture is.

**Pull request merged**